pkg/config/remote/client.go

```diff
@@ -363,9 +363,10 @@ func (c *Client) newUpdateRequest() (*pbgo.ClientGetConfigsRequest, error) {
 	pbConfigState := make([]*pbgo.ConfigState, 0, len(state.Configs))
 	for _, f := range state.Configs {
 		pbConfigState = append(pbConfigState, &pbgo.ConfigState{
-			Id:      f.ID,
-			Version: f.Version,
-			Product: f.Product,
+			Id:         f.ID,
+			Version:    f.Version,
+			Product:    f.Product,
+			ApplyState: uint64(f.ApplyStatus.State),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏It's good that we report the configuration status, but is it required to fix the bugs in this PR?